peachjam/static/js/report_document_issue.js

```diff
@@ -0,0 +1,35 @@
+function handleResponse (reportForm, response, url) {
+  const modalLabel = document.getElementById('reportModalLabel');
+  modalLabel.textContent = '';
+  reportForm.innerHTML = '';
+  reportForm.innerHTML = response;
+
+  setTimeout(() => {
+    const modal = document.getElementById('reportModal');
+    modal.remove();
+    window.location.href = url;
+  }, 1000);
+}
```

Comment:
@nickmwangemi In my opinion, its better to allow the user to close the modal on their own. It will ensure that they read the success message and acknowledge that the message is sent, perhaps you could render a close button on the modal instead of closing it automatically?

Reply:
@nickmwangemi I just saw that the close button is present in the modal. So I think all you need to do is remove this bit of code. you could then put the submit button on the same row as the close button. The default styles by bootstrap look quite good

https://getbootstrap.com/docs/5.2/components/modal/
![image](https://github.com/laws-africa/peachjam/assets/25079238/85c803da-04f3-42bb-95e7-5c7a2fa86f5e)